APE_coordinated_pkg.rst

```diff
@@ -0,0 +1,235 @@
+An Updated Model for the Affiliated Package Ecosystem
+-----------------------------------------------------
+
+author: Erik Tollerud, 2017 Astropy Coordination Meeting Attendees
```

Comment:
@eteq - I would just list yourself - as long as you give credit to the coordination meeting attendees in the main text, you don't need to include them/us as authors. For one, this includes the coordination committee itself which makes the task of accepting this APE tricky, and second you were the actual author of the APE. In a few years people won't even know who *2017 Astropy Coordination Meeting Attendees* was - I don't think it makes sense to refer to vague collections of people in the author list.